Consts.py

```diff
@@ -45,3 +45,19 @@
 
 # degree frequency at which to generate lidar data
 degree_freq = 1
+
```

Comment:
get rid of this stuff?

Reply:
All of the added constants? They're used for the new methods that we implemented though

Reply:
makes sense, thanks